db/schema.rb

```diff
@@ -14,7 +14,6 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "citext"
-  enable_extension "pg_stat_statements"
```

Comment:
I think these changes are unintentional - would you mind removing them from this PR? thanks!

Reply:
Yea sure those were auto generated while setting up. should I remove both the changes in this file?

Reply:
Yes please! This happens sometimes when setting up locally (just a symptom of switching between different branches), and the changes can safely be discarded 👍

Reply:
Done with the changes, could you review it again?